src/main/java/jarvis/task/Todo.java

```diff
@@ -0,0 +1,12 @@
+package jarvis.task;
+
+public class Todo extends Task{
```

Comment:
Here, you could consider including a space before the`{`, as with the other classes.